build/jquery.bootgrid.js

```diff
@@ -493,9 +493,9 @@
                     {
                         var commandList = {
                             first: 1,
-                            prev: that.current - 1,
-                            next: that.current + 1,
-                            last: that.totalPages
+                            prev: parseInt(that.current) - 1,
```

Comment:
These changes need to be added to the internal.js file - this file is built by Grunt from all the files in the src directory :)